src/tests/core/test_management_commands.py

```
@@ -0,0 +1,241 @@
+
+    def test_auto_backup_creates_backup(self, db):
+        """The auto_backup command creates a backup file."""
+        with TemporaryDirectory() as tmpdir:
+            out = StringIO()
+            call_command("auto_backup", f"--output={tmpdir}", "--keep=3", stdout=out)
+
+            # A backup should be created
+            backups = list(Path(tmpdir).glob("datakult_backup_*.tar.gz"))
+            assert len(backups) == 1
+
+    def test_auto_backup_rotates_old_backups(self, db):
```

Ruff (0.14.10)

Unused method argument: `db`

(ARG002)